client/src/components/WeatherCard.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import './WeatherCard.css'
+
+function convertUnixTimestampToCityTime(unixTimestamp, utcOffsetInSeconds) {
+    const utcSeconds = unixTimestamp + utcOffsetInSeconds;
+    const cityDate = new Date(utcSeconds * 1000);
+
+    return cityDate.toLocaleTimeString(undefined, {
+        timeZone: 'UTC',
+        hour: 'numeric',
+        minute: 'numeric'
+    });
+}
+
+
+
+function WeatherCard({ data, city }) {
+     
+    if (data && city) {
+        return (
+            <div className='weatherCard'>
+                <h3>
+                    <span>{data.name}, {data.sys.country}</span>
+                </h3>
+                <h3>
+                    <span>{Math.round((data.main.temp-273.15) * 9/5 + 32)} Fahrenheit, {Math.round((data.main.temp-273.15))} Celsius </span>
+                </h3>
+                <h3>
+                    <span> Feels like: {Math.round((data.main.feels_like-273.15) * 9/5 + 32)} Fahrenheit, {Math.round((data.main.feels_like-273.15))} Celsius </span>
```

Comment:
This is all fine, but me being picky again - if it was me I would do all these calculations and rounding up above the return line in the function and assign the results to variables. Then here in the rendering code just pass those variables. Will make the code look cleaner and easier to read.